src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -6524,11 +6524,11 @@ public void ParseAdditionalFile()
         private static int OccurrenceCount(string source, string word)
         {
             var n = 0;
-            var index = source.IndexOf(word);
+            var index = source.IndexOf(word, StringComparison.Ordinal);
```

Comment:
I'm not adding it right now. Current clients of this code do not provide an argument for it.